src/dll/edbgEcmdDllInfo.C

```diff
@@ -41,10 +41,10 @@ uint32_t dllQueryDllInfo(ecmdDllInfo & o_dllInfo) {
   o_dllInfo.dllProduct = "POWER9";
   //o_dllInfo.dllType = "edbg";
   //o_dllInfo.dllProduct = "OpenPOWER";
-  o_dllInfo.dllProductType = GIT_COMMIT_REV;
+  //o_dllInfo.dllProductType = GIT_COMMIT_REV;
   o_dllInfo.dllEnv = ECMD_DLL_ENV_HW;  
 
-  o_dllInfo.dllBuildDate = BUILD_DATE;
```

Comment:
Same as above

Reply:
Yes. will do

Reply:
done

Reply:
> > Are all these related ecmd-pdbg compilation? There are ways in meson build but probably it will take some time to deploy. I can add those in future PR's as this will not break existing build progress
> 
> Yes, those defines are related to ecmd-pdbg build. Without them, the compile will fail. They are just defines, so most of them could probably moved into a common header file (like `src/common/edbgCommon.H`). That would get them out of the build process specifics and into a common place either build could utilize.

We might still need to redirect the defines like said above.. meson currently uses global flags